frontends/web/src/routes/exchange/exchange.tsx

```diff
@@ -134,7 +134,13 @@ export const Exchange = ({ code, accounts, deviceIDs }: TProps) => {
           )}
         </Dialog>
         <div className="innerContainer scrollableContainer">
-          <Header title={<h2>{title}</h2>} />
+          { activeTab === 'spend' ? (
+            <Header title={<h2>{t('generic.spend', {
+              context: hasOnlyBTCAccounts ? 'bitcoin' : 'crypto'
+            })}</h2>}/>
+          ) : (
+            <Header title={<h2>{title}</h2>}/>
+          )}
```

Comment:
nit: an alternative could be to use 1 `<Header ` element and move the condition into title attribute.